.github/workflows/stale.yml

```diff
@@ -0,0 +1,20 @@
+name: Stale PR Reminder
+
+on:
+  schedule:
+    - cron: '0 12 * * *' # Run every day at 12:00 UTC
+
+jobs:
+  stale-reminder:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Determine staleness
+      uses: actions/stale@v5
+      with:
+        stale-pr-message: 'This PR is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in next 60 days.'
+        close-pr-message: 'This PR was closed because it has been stalled for 90 days with no activity. You could reopen the PR if you still would like to work on it, and please make sure to have some kind of activity in your PR , as leaving it stalled for too long would result in the PR being closed.'
```

Comment:
```suggestion
        close-pr-message: 'This PR was closed automatically because there hasn't been any activity for 90 days. You can reopen the PR if you would like to continue to work on it.'
```